Doc/library/socketserver.rst

```diff
@@ -81,20 +81,20 @@ synchronous servers of four types::
    +------------+
    | BaseServer |
    +------------+
+         ^
          |
-         v
    +-----------+        +------------------+
-   | TCPServer |------->| UnixStreamServer |
+   | TCPServer |<-------| UnixStreamServer |
    +-----------+        +------------------+
+         ^
          |
-         v
    +-----------+        +--------------------+
-   | UDPServer |------->| UnixDatagramServer |
+   | UDPServer |<-------| UnixDatagramServer |
    +-----------+        +--------------------+
 
 Note that :class:`UnixDatagramServer` derives from :class:`UDPServer`, not from
 :class:`UnixStreamServer` --- the only difference between an IP and a Unix
-stream server is the address family, which is simply repeated in both Unix
+server is the address family, which is simply repeated in both Unix
```

Comment:
Why is this change being made?

Reply:
Because the difference is also valid for IP and Unix *datagram* servers, not only between IP and Unix *stream* servers.

Reply:
IMO I find this weird to read without "stream". A possibility would be to say something like "_the only difference between IP and a Unix stream or datagram server..."_. Perhaps @merwok has a better suggestion.

Reply:
+1, I had the same feeling, though the phrase ‘Unix server’ is already used at the next line (line 98). Updated.